builtin/plugin/backend.go

```diff
@@ -237,3 +237,53 @@ func (b *PluginBackend) HandleExistenceCheck(ctx context.Context, req *logical.R
 	}
 	return checkFound, exists, err
 }
+
+// Initialize is a thin wrapper implementation of Initialize that includes automatic plugin reload.
+func (b *PluginBackend) Initialize(ctx context.Context, req *logical.InitializationRequest) error {
+	b.RLock()
+	canary := b.canary
+
+	// Lazy-load backend
+	if !b.loaded {
```

Comment:
Can you consolidate the logic that's there here and in HandleRequest and HandleExistenceCheck instead of duplicating it a third time?